brave-unbreak.txt

```diff
@@ -169,6 +169,9 @@
 ! Fix facebook logins on messenger.com https://github.com/brave/brave-browser/issues/4173
 @@||facebook.com/login/$domain=messenger.com
 @@||connect.facebook.net^$domain=messenger.com
+! Anti-adblock: techsolutiontips.com
+@@||googlesyndication.com/pagead/js/r20190612/r20190131/show_ads_impl.js$domain=techsolutiontips.com
```

Comment:
This rule looks specific to the date you visited the page on.  Needs to be generalized